docs/API.md

```diff
@@ -241,33 +271,26 @@ Type: `Object`
 
 Options, that are passed to [`gaze`](https://github.com/shama/gaze).
 
-#### cb(event)
-Type: `Function`
+#### tasks
+Type: `Array`, `Function` or `String`
 
-Callback to be called on each change.
+A task name, a function or an array of either to run when a file changes.
```

Comment:
make not about the reason to use `watcher.on('change')` if the gaze `event` is needed